lib/organisation_importer.rb

```diff
@@ -24,13 +24,16 @@ def create_or_update_organisation(organisation)
     if existing_tag.present?
       logger.info "Found existing tag: #{tag_id}"
 
-      if existing_tag.title != organisation.title
-        if existing_tag.update_attributes(title: organisation.title)
-          logger.info "Updated title to '#{existing_tag.title}'"
-        else
-          log_error_and_notify_airbrake(organisation,
-                                        "Could not update title: #{existing_tag.errors.full_messages}")
+      existing_tag.title = organisation.title
+      existing_tag.content_id = organisation.details.content_id
+
+      if existing_tag.save
+        if existing_tag.changes.any?
```

Comment:
I think this would be clearer:`if existing_tag.save && if existing_tag.changes.any?`